theme/_sass/custom.scss

```diff
@@ -52,3 +89,15 @@ blockquote p {
 .table-responsive {
   margin-bottom: 1em;
 }
+
+#documents {
+  list-style: none;
+}
+
+.nav li {
```

Comment:
Sass allows you to do some cool nesting which helps readability and conciseness for CSS: https://css-tricks.com/the-sass-ampersand/